### PR TITLE
Mergeup from master. This time with feeling.

### PR DESCRIPTION
**Pull request by e-gris**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -238,6 +242,10 @@ this is a final version.
 [0.99.2]: https://github.com/puppetlabs/packaging/compare/0.99.1...0.99.2
 [0.99.1]: https://github.com/puppetlabs/packaging/compare/0.99.0...0.99.1
 [0.99.0]: https://github.com/puppetlabs/packaging/compare/0.6.2...0.99.0
+=======
+[Unreleased]: https://github.com/puppetlabs/packaging/compare/0.6.8...master
+[0.6.8]: https://github.com/puppetlabs/packaging/compare/0.6.7...0.6.8
+>>>>>>> master
```
Comment (mwaggett) [inline CHANGELOG.md]: merge conflict 😊 
you'll want to keep the [0.6.8] line, but not the 0.6.8...master

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+## [0.6.8] - 2018-08-14
```
Comment (sarameisburger) [inline CHANGELOG.md]: We'll wanna move this down here https://github.com/puppetlabs/packaging/pull/886/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR137

Comment (mwaggett) [reply CHANGELOG.md]: oh yeah good call

Comment (shrug) [inline CHANGELOG.md]: Move me down to the 0.6 section!

**Pull request merged**